spec/defines/sensu_check_spec.rb

```diff
@@ -105,7 +105,8 @@
         :handle              => true,
         :publish             => true,
         :auto_resolve        => true,
-        :ttl                 => 30
+        :ttl                 => 30,
+        :ttl_status          => 1
```

Comment:
what happens if ttl_status is a non integer and not `'absent'` ?

Reply:
if is set to absent then have default value 1 (which mean warning)
I don't know what happens if will be set to string, probably sensu client die.
Probably we should check also ttl, interval refresh etc

Reply:
I meant, what will Puppet do? We want some checks to ensure that Puppet errors appropriately.